src/vs/workbench/contrib/extensions/electron-browser/extensionsViewlet.ts

```diff
@@ -435,7 +435,11 @@ export class ExtensionsViewlet extends ViewContainerViewlet implements IExtensio
 
 	protected saveState(): void {
 		const value = this.searchBox.getValue();
-		this.searchViewletState['query.value'] = value;
+		if (value.indexOf('@installed') >= 0) {
```

Comment:
May be you should use the following to check if it is an installed query or not

https://github.com/Microsoft/vscode/blob/68674a96456733ba8e4f62002d5c473be918d5f4/src/vs/workbench/contrib/extensions/electron-browser/extensionsViews.ts#L704

Reply:
fixed it